generation/diff_files.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+#run this script from google-cloud-java directory
+# google-cloud-java$ ./generation/diff_files.sh
+# not using set -e because failing diff command is ok
+# latest repositories Id -> comgooglecloud-5570, comgoogleapi-5038, comgoogleanalytics-1052
+# this script must be run on a branch which does not have any snapshot versions in it
+# Run the stage job for google-cloud-java, on a branch which does not have any snapshot versions in it.
+# search the stage job logs for the latest-repo-ids and edit them below under `cloudRepoId`, `apiRepoId`, `analyticsRepoId`
+# Output of this script are 2 files ->
+# 1. diff-files-summary.txt : This will show success for artifacts which have same files on maven-central and sonatype,
+# and if they differ, it will show a diff failure message along with the names of the files that differ.
+# 2. total-diff.txt : For every artifact, this will show 4 things:
+      # a. Sonatype directory URL
+      # b. Files that exist on sonatype (with version omitted, since we only care about the file generated)
+      # c. Maven directory
+      # d. Files that exist on Maven (with version omitted, since we only care about the file generated)
+# This script calls ./generation/diff_directory.sh for every pair, but you do not need to do anything. You just need to run this script.
+# Search total-diff.txt for any artifact, and it will show you a complete scenario, what all files exist etc etc.
+
+set -x
```

Comment:
Can you remove '-x'? When someone needs "-x", they can do it with "bash -x ..."

Reply:
removed.